exe/Parser/Text.hs

```diff
@@ -507,6 +548,51 @@ parseDetailedChar line =
     (_sLower, line12) = span (/= ';') (tail line11)
     _sTitle = tail line12
 
+isDivider :: String -> Bool
+isDivider x = x == "# ================================================"
+
+-- XXX Should replace readLinesFromHandle
+readLinesFromHandle1 :: IsStream t => Handle -> t IO String
+readLinesFromHandle1 h =
+    Stream.unfold Handle.read h & Unicode.decodeUtf8 & Unicode.lines Fold.toList
+
+readLinesFromFile :: IsStream t => String -> t IO String
+readLinesFromFile file =
+    Stream.bracket
+        (Sys.openFile file Sys.ReadMode)
+        Sys.hClose
+        readLinesFromHandle1
```

Comment:
Can you instead use something from the `Streamly.Internal.FileSystem.File` module?